ui/src/consts.js

```diff
@@ -1,3 +1,5 @@
 export const REST_URL = '/app/api'
 
 export const M3U_MIME_TYPE = 'audio/x-mpegurl'
+
+export const AUTO = 'AUTO'
```

Comment:
Please rename to a more descriptive const name, like `AUTO_THEME_ID`

Reply:
Done!